Gems/Terrain/Code/Source/Components/TerrainWorldRendererComponent.cpp

```diff
@@ -85,15 +99,65 @@ namespace Terrain
+                    ->DataElement(AZ::Edit::UIHandlers::Slider, &ClipmapConfiguration::m_macroClipmapMaxResolution, "Macro clipmap max resolution", "The resolution of the highest resolution clipmap in the stack.")
+                        ->Attribute(AZ::Edit::Attributes::Min, 2.0f)
+                        ->Attribute(AZ::Edit::Attributes::SoftMin, 2.0f)
+                        ->Attribute(AZ::Edit::Attributes::Max, 10.0f)
+                        ->Attribute(AZ::Edit::Attributes::SoftMax, 10.0f)
+                    ->DataElement(AZ::Edit::UIHandlers::Slider, &ClipmapConfiguration::m_detailClipmapMaxResolution, "Detail clipmap max resolution", "The resolution of the highest resolution clipmap in the stack.")
+                        ->Attribute(AZ::Edit::Attributes::Min, 512.0f)
+                        ->Attribute(AZ::Edit::Attributes::SoftMin, 512.0f)
+                        ->Attribute(AZ::Edit::Attributes::Max, 2048.0f)
+                        ->Attribute(AZ::Edit::Attributes::SoftMax, 2048.0f)
+                    ->DataElement(AZ::Edit::UIHandlers::Slider, &ClipmapConfiguration::m_macroClipmapScaleBase, "Macro clipmap scale base", "The scale base between two adjacent clipmap layers.")
+                        ->Attribute(AZ::Edit::Attributes::Min, 2.0f)
+                        ->Attribute(AZ::Edit::Attributes::SoftMin, 2.0f)
+                        ->Attribute(AZ::Edit::Attributes::Max, 4.0f)
+                        ->Attribute(AZ::Edit::Attributes::SoftMax, 4.0f)
+                    ->DataElement(AZ::Edit::UIHandlers::Slider, &ClipmapConfiguration::m_detailClipmapScaleBase, "Detail clipmap scale base", "The scale base between two adjacent clipmap layers.")
+                        ->Attribute(AZ::Edit::Attributes::Min, 2.0f)
+                        ->Attribute(AZ::Edit::Attributes::SoftMin, 2.0f)
+                        ->Attribute(AZ::Edit::Attributes::Max, 4.0f)
+                        ->Attribute(AZ::Edit::Attributes::SoftMax, 4.0f)
+                    ->DataElement(AZ::Edit::UIHandlers::Slider, &ClipmapConfiguration::m_macroClipmapMarginSize, "Macro clipmap margin size", "The margin of the clipmap where the data won't be used.")
+                        ->Attribute(AZ::Edit::Attributes::Min, 1u)
+                        ->Attribute(AZ::Edit::Attributes::SoftMin, 1u)
+                        ->Attribute(AZ::Edit::Attributes::Max, 8u)
+                        ->Attribute(AZ::Edit::Attributes::SoftMax, 8u)
+                    ->DataElement(AZ::Edit::UIHandlers::Slider, &ClipmapConfiguration::m_detailClipmapMarginSize, "Detail clipmap margin size", "The margin of the clipmap where the data won't be used.")
+                        ->Attribute(AZ::Edit::Attributes::Min, 1u)
+                        ->Attribute(AZ::Edit::Attributes::SoftMin, 1u)
+                        ->Attribute(AZ::Edit::Attributes::Max, 8u)
+                        ->Attribute(AZ::Edit::Attributes::SoftMax, 8u)
```

Comment:
I think the description needs more information here. Just saying the data won't be used will make people wonder why have a margin at all. Am I right in guessing that a larger margin means fewer but larger updates as you move around the world? When we move to compressed textures this will need to be at least 4, so should probably make that the hard min so we don't need to change this later.

Reply:
I have hidden the configuration.